OPERATING.md

```diff
@@ -22,6 +22,59 @@ yyyy-mm-dd HH:mm:ss.SSS Level=DEBUG Logger=u.n.a.p.t.s.BundleMapperService Conve
 
 ## Database requirements
 
+* The adaptor requires a [PostgreSQL] database
+* The adaptor stores the identifiers, status, and metadata for each patient transfer
+* The adaptor uses the database as a source of SNOMED information
+* Deleting the database, or its records will cause any in-progress transfers to fail
+* The database can be used to monitor for any failed or incomplete transfers
```

Comment:
Should we also mention the facade here as it's the initial way to see a transfer has failed or is incomplete? We could just link to the  README as it's already described there.